spec/integration/manifest.spec.yml

```diff
@@ -225,13 +225,21 @@ spec:
             enum:
               - action
               - next_step
+          appId:
+            description: The identifier of the Kibana app to navigate to. Set this field when `internal` is true.
```

Comment:
How does a package developer know the app to navigate to? Do they need to get it from the URL?

Reply:
yes it's basically the plugin, it's usually in the url